jupyterhub/assignments/intake-dashboard/report_districts.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "%load_ext sql\n",
+    "%config SqlMagic.feedback = False\n",
+    "%config SqlMagic.displaylimit = 0\n",
+    "%config SqlMagic.displaycon = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "start = datetime.datetime.now() - datetime.timedelta(days=90)\n",
+    "end = datetime.datetime.now()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql report_districts <<\n",
+    "SELECT \n",
+    "    r.district as district,\n",
+    "    r.primary_statute as classification,\n",
+    "    r.primary_complaint as complaint,\n",
+    "    r.location_state as state,\n",
+    "    COUNT(*) count\n",
+    "FROM cts_forms_report r\n",
+    "WHERE district notnull AND primary_complaint notnull\n",
+    "GROUP BY complaint, district, classification, state\n",
+    "HAVING COUNT(*) > 1\n",
+    "ORDER BY count desc"
```

Comment:
this (and your other query) should be very efficient, nice work!

Only suggestion - do you want to filter by the `start` and `end` date filters from above?